src/database/seeds/1743608138837_test_users.ts

```diff
@@ -0,0 +1,99 @@
+import type { Kysely, Selectable } from 'kysely'
+import { DB, MemberUser, User } from '@/database/types'
+import { db } from '@/database'
+import { v7 as uuidv7 } from 'uuid'
+
+export async function seed(db: Kysely<DB>): Promise<void> {
```

Comment:
we need to check an env var check so this doesn't run in prod, maybe something like:
```suggestion
export async function seed(db: Kysely<DB>): Promise<void> {
    if (process.env.NO_TESTING_DATA) return;
```